content/using/install.md

```diff
@@ -118,9 +122,16 @@ http: live (insecure, loopback) on 12321
 
 When your ship is finished booting, you will see either the `~sampel_commet:dojo>` or `~sampel_commet:chat-cli/` prompt. If you're seeing `:chat-cli` press `Ctrl-x` to switch into Dojo. 
 
-To exit Urbit, use `Ctrl-d`. 
+To exit Urbit, use `control-d` or enter `|exit` into Dojo.
```

Comment:
`Ctrl+D` is actually clearest. But be consistent with the `Ctrl-x` right above it...

Reply:
I'll make the change